src/main/kotlin/de/bund/digitalservice/useid/tenant/Tenant.kt

```diff
@@ -0,0 +1,10 @@
+package de.bund.digitalservice.useid.tenant
+
+class Tenant {
+    lateinit var id: String
+    lateinit var apiKey: String
+    lateinit var refreshAddress: String
+    var dataGroups: List<String> = emptyList()
+    var allowedHosts: List<String> = emptyList()
+    var cspNonce: String = ""
```

Comment:
Why default empty String here and not `lateinit` as well?

Reply:
Because this is not being injected by the application.yaml, thus a default value is better because it will run into problems when tried to be accessed. 
My assumption.

Reply:
But don't we want to run into problems if we try to access it but it is not set? Otherwise the nonce would have no value and we would not even notice it. Only the user would since the scripts would not be executed.